src/sentry/models/organizationmember.py

```diff
@@ -62,9 +63,19 @@ def get_contactable_members_for_org(self, organization_id: int) -> QuerySet:
 
     def delete_expired(self, threshold: int) -> None:
         """Delete un-accepted member invitations that expired `threshold` days ago."""
+
+        # if SCIM is enabled, let SCIM control creation, modification, and deletion of users
+        orgs_without_scim = [
+            org_id
+            for org_id in AuthProvider.objects.exclude(flags__gt=1).values_list(
```

Comment:
`flags__gt=1` will fail if an additional flag is added to `AuthProvider` and set to true. Take a look at how we use `F()` here https://github.com/getsentry/sentry/blob/3dd203a50ee2828fe66fde89057cf63ad1dfb603/src/sentry/web/client_config.py#L73-L75